Makefile

```diff
@@ -236,7 +236,7 @@ travis_rosetta_checker:
 	TEST_REPO_BRANCH='master'
 	bash ./scripts/travis_rosetta_checker.sh
 
-debug_external: clean
+debug_external:
```

Comment:
we use make debug-ext which also call the the script to build the external validator 

hence let's remove this completely and delete https://github.com/harmony-one/harmony/blob/dev/Makefile#L57

Reply:
done in the last commit